masterlist.yaml

```diff
@@ -5833,6 +5833,17 @@ plugins:
       - 'Farm_DLCNukaWorld.esp'
       - 'Farm_DLCworkshop03.esp'
 
+  - name: 'Winter Overhaul.esp'
```

Comment:
May as well add the location for future maintenance.

```suggestion
  - name: 'Winter Overhaul.esp'
    url: [ 'https://www.nexusmods.com/fallout4/mods/28665/' ]
```

Reply:
Should be solved. The original PR had it, dunno what I did there